portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,44 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+    const greetings =
+        ['Courage is your greatest present need.', 'Extreme fear can neither fight nor fly.', 'Your business will assume vast proportions.'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    // Pick a random greeting.
+    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+    // Add it to the page.
+    const greetingContainer = document.getElementById('greeting-container');
+    greetingContainer.innerText = greeting;
 }
+
+var slideIndex = 1;
```

Comment:
Nit: I would use 0 as the starting index since it's easier for assignment later on (instead of keep using `slideIndex  - 1`).